srcpkgs/eb/patches/0001-cross.patch

```diff
@@ -0,0 +1,111 @@
+From 46b9fd687125d963b8e3b675f272c5348bdb28dd Mon Sep 17 00:00:00 2001
+From: Nathan Owens <ndowens@artixlinux.org>
+Date: Mon, 4 Jan 2021 09:53:39 -0600
+Subject: [PATCH] cross
+
+---
+ m4/largefile.m4    |  4 ++--
+ m4/sockinttypes.m4 | 16 ++++++++--------
+ 2 files changed, 10 insertions(+), 10 deletions(-)
+
+diff --git m4/largefile.m4 m4/largefile.m4
+index 7d98cae..f033367 100644
+--- m4/largefile.m4
++++ m4/largefile.m4
+@@ -32,7 +32,7 @@ dnl *
+ AC_DEFUN([eb_SYS_LARGEFILE], [dnl
+ AC_REQUIRE([AC_SYS_LARGEFILE])
+ AC_CACHE_CHECK([for ll modifier of printf], ac_cv_func_printf_ll,
+-[AC_RUN_IFELSE([
++[AC_COMPILE_IFELSE([
```

Comment:
This is incorrect patch.

Reply:
Any pointer or reading material to understand this more? I thought replacing AC_RUN* with AC_COMPILE_IFELSE was ok to do?